cli.sh

```diff
@@ -856,15 +876,16 @@ add_shared_hook_repo() {
         [ -f "$(pwd)/.githooks/.shared" ] &&
             echo "" >>"$(pwd)/.githooks/.shared"
 
-        echo "# Added on $(date)" >>"$(pwd)/.githooks/.shared" &&
-            echo "$SHARED_REPO_URL" >>"$(pwd)/.githooks/.shared" &&
-            echo "The new shared hook repository is successfully added" &&
-            echo "  Do not forget to commit the change!" &&
-            return
-
-        echo "! Failed to add the new shared hook repository" >&2
-        exit 1
+        if ! echo "# Added on $(date)" >>"$(pwd)/.githooks/.shared" ||
```

Comment:
There seem to be a few changes like this for bare repos.
I'd prefer leaving the previous print chains as-is, and perhaps add a new function: `echo_if_not_bare_repo ${msg}`
That could then suppress printing the message if not needed for bare repos.

On a side note: does printing these messages on a bare repo cause any issues?